docs/man/keepassxc-cli.1.adoc

```diff
@@ -251,6 +256,13 @@ The same password generation options as documented for the generate command can
   Available choices are xml or csv.
   Defaults to xml.
 
+=== Import options
+*-k*, *--set-key-file* <__path__>::
+  Set the key file for the database.
+
+*-p*, *--set-password*::
+  Set a password for the database.
```

Comment:
@Colfenor at that point, the only difference between the Create options and the Import options is that we allow to set the decryption time in `db-create`. How hard would it be to reuse the same database creation code for `db-create` and `import`? This way we would make sure we always support the same options in both commands, and we could also merge the documentation in the manpage under a `Create and Import options` header, like we do for other commands that same options

Reply:
Hi @louib ! Yes I totally agree, to provide a consistent behavior of both commands I added the code of the decryption time parameter option to the import command. I tested it locally and it works. Moreover the manpage is now updated aswell as the cli-tests. Ready for review :)

Reply:
The code for the database creation is duplicated in `Import` and `Create`, no? Could we create a static function in `Create` that takes the args parser and return a db, and use that function in both `Create` and `Import`?

Is `db.setKdf(KeePass2::uuidToKdf(KeePass2::KDF_ARGON2));` what we do when a db is created from the UI?

Reply:
> The code for the database creation is duplicated in `Import` and `Create`, no? Could we create a static function in `Create` that takes the args parser and return a db, and use that function in both `Create` and `Import`?

The first part of the database creation is duplicated exactly however, later on in the **Create** cmd the db object is initialized as `QSharedPointer<Database> db(new Database);` on the heap whereas, in the **Import** cmd we have a simple `Database db;` stack object which gets operated on.

I'm not entirely sure on why there is a difference, both ways seem to work so far from the cli.
However maybe we have a bug here ?
 
> Is `db.setKdf(KeePass2::uuidToKdf(KeePass2::KDF_ARGON2));` what we do when a db is created from the UI?

Yes exactly :) 

https://github.com/keepassxreboot/keepassxc/blob/develop/src/gui/dbsettings/DatabaseSettingsWidgetEncryption.cpp#L88

Reply:
@Colfenor 

If we're defaulting to ARGON2 in the GUI I guess we should be doing the same in the CLI. We could also created a `DEFAULT_KDF` constant available to both and use that.

> The first part of the database creation is duplicated exactly, however later on in the Create cmd the db object is initialized as QSharedPointer<Database> db(new Database); on the heap whereas, in the Import cmd we have a simple Database db; stack object which gets operated on.

I feel like both should be handled exactly the same. Not sure which one is best tho. If both work right now I'd say pick one, and if we change in the future we'll only have to change it in the static function.

Reply:
@louib 

> If we're defaulting to ARGON2 in the GUI I guess we should be doing the same in the CLI. 
Ok, changed it.

> I feel like both should be handled exactly the same. Not sure which one is best tho. If both work right now I'd say pick one, and if we change in the future we'll only have to change it in the static function.

Alright, I chose to create the db on the stack. The cli-tests still work and I tested it locally.